px-map-tile-layer-wmts.html

```diff
@@ -0,0 +1,185 @@
+<link rel="import" href="../polymer/polymer.html"/>
+<link rel="import" href="px-map-behavior-tile-layer.html"/>
+
+<!--
+This subcomponent displays WMTS services as tile layers on the map. It is a simple
+wrapper around the Leaflet mapping library's L.tileLayer.wmts layer. It has also included a wmts plugin leaflet-tilelayer-wmts that supports Leaflet. Find the plugin and leaflet plugin details here. [on the github website](https://github.com/mylen/leaflet.TileLayer.WMTS).
+
+### Usage
+
+    <px-map>
+      <px-map-tile-layer-wmts 
+        url="http://example.com/cgi-bin/wmts/map.cgi" 
+        layer="WorldTimeZones" 
+        format="image/png" 
+        attribution="Example data (c) 2017 Example Corp."> 
+      </px-map-tile-layer-wmts>
+    </px-map>
+
+@element px-map-tile-layer-wmts
+@blurb Displays WMTS services as tile layers
+@homepage index.html
+@demo index.html
+-->
+
+<script src="leaflet-tilelayer-wmts.js"></script>
+
+<dom-module id="px-map-tile-layer-wmts">
+  <style>
+    :host { display: none }
+  </style>
+  <template></template>
+</dom-module>
+
+<script>
+  Polymer({
+    is: 'px-map-tile-layer-wmts',
+    behaviors: [PxMapBehavior.Layer],
+
+    properties: {
+      /**
+       * Base URL of the WMTS service.
+       */
+      url: {
+        type: String,
+        observer: 'shouldUpdateInst'
+      },
+
+      /**
+       * WMTS layers to show. Required to show the WMTS
+       * tile layers.
+       */
+      layer: {
+        type: String,
+        observer: 'shouldUpdateInst'
+      },
+
+      /**
+       * WMTS request to get the Tile.
+       */
+      request: {
+        type: String,
+        observer: 'shouldUpdateInst',
+        value: 'gettile'
+      },
+
+      /**
+       * Comma-separated list of WMTS styles.
+       */
+      style: {
+        type: String,
+        observer: 'shouldUpdateInst',
+        value: 'default'
+      },
+
+      /**
+       * WMTS image format to use (use 'image/png' for layers).
```

Comment:
In the doc we say to use `image/png` for layers then default to `image/jpeg`, which one is the correct default?